### PR TITLE
Karmeshyadav/small changes

### PR DESCRIPTION
**Pull request by ykarmesh**
## Types of changes

- Docs change / refactoring / dependency upgrade
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by yvsriram — APPROVED** [review]

File: habitat-lab/habitat/config/benchmark/rearrange/cat_gaze.yaml [hunk 1]
```diff
@@ -44,10 +44,12 @@ habitat:
              height: 256
              width: 256
              hfov: 42
+             position: [0,1.31,0]
            head_panoptic_sensor:
              height: 256
              width: 256
              hfov: 42
+             position: [0,1.31,0]
```
Comment (yvsriram) [inline habitat-lab/habitat/config/benchmark/rearrange/cat_gaze.yaml:52]: I assume you used something like this before to set the height. I never tested this before

File: habitat-lab/habitat/config/benchmark/rearrange/cat_place.yaml [hunk 2]
```diff
@@ -42,12 +42,16 @@ habitat:
              height: 256
```
Comment (yvsriram) [inline habitat-lab/habitat/config/benchmark/rearrange/cat_place.yaml]: Do we want to start training with the same aspect ratio?

File: habitat-lab/habitat/config/benchmark/rearrange/cat_gaze.yaml (hunk 1 above)
Comment (ykarmesh) [reply habitat-lab/habitat/config/benchmark/rearrange/cat_gaze.yaml:52]: Yes I have used this in Hab1.0 task. But now thinking more, I dont know if the URDF plays any role in finding out the height of the camera for Hab2.0 tasks.

File: habitat-lab/habitat/config/benchmark/rearrange/cat_place.yaml (hunk 2 above)
Comment (ykarmesh) [reply habitat-lab/habitat/config/benchmark/rearrange/cat_place.yaml]: Ok let me post another update for that and then we can merge this

**Pull request merged**